### Files look OK. In CLHS this topis is *PRINT-RIGHT-MARGIN*
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:variables:star-print-right-margin-star.txt	2017-04-27 21:45:43.000000000 +0100
+++ saved-pages-clus/cl:variables:star-print-right-margin-star.txt	2017-04-27 21:45:43.000000000 +0100
@@ -1,18 +1,38 @@
 Variable *PRINT-RIGHT-MARGIN* 
  
 Value Type
  
 a non-negative integer, or nil. 
 Initial Value
  
 nil. 
 Description
  
 If it is non-nil, it specifies the right margin (as integer number of ems) to use when the pretty printer is making layout decisions. 
 If it is nil, the right margin is taken to be the maximum line length such that output can be displayed without wraparound or truncation. If this cannot be determined, an implementation-dependent value is used. 
-Examples None.
  
-See Also None.
+Examples
+None.
+See Also
+None.
  
 Notes
  
